Harvest.Web/ClientApp/src/Requests/RequestContainer.tsx

```diff
@@ -147,166 +148,178 @@ export const RequestContainer = () => {
   }
 
   return (
-    <div className={originalProject && "card-wrapper"}>
-      {originalProject !== undefined && (
-        <ProjectHeader
-          project={originalProject}
-          title={"Original Field Request #" + (originalProject?.id || "")}
-        />
+    <div>
+      {projectId && (
```

Comment:
@wongband This works but to be consistent you should use `originalProject !== undefined` as your check